num_scouts: 1
num_excavators: 0
num_haulers: 1
clock_max_time_sec: 7200
planner:
  planning_timeout_sec: 30
  planner_type: "type1"
  num_nodes: 500
  num_iterations: 5
cost_function:
  type: 0
  params: [0.0, 1, 5.0]
demo: false
